tests/modules/nf-core/gem2/gem2bedmappability/nextflow.config

```diff
@@ -0,0 +1,12 @@
+process {
+
+    publishDir = { "${params.outdir}/${task.process.tokenize(':')[-1].tokenize('_')[0].toLowerCase()}" }
+
+    withName: GEM2_GEMINDEXER {
+        publishDir = [enabled:false]
+    }
+
+    withName: GEM2_GEMMAPPABILITY {
+        publishDir = [enabled:false]
```

Comment:
Why is the publishDir disabled?

Reply:
Because I'm not testing this tool here :)